### ISSUE TITLE
Metaworld offline data

### ISSUE THREAD

**Issue by kevin-xuan** [eric-mitchell/macaw#3]
Hi, I'm interested in your work and thanks for your sharing code. In the paper, you conduct ML45 experiments on metaworld benchmarks, however,  the relevant offline data are not provided by you. Could you please offer the ML45 offline data? 
Thanks for your reply!

**Comment by shu-bamma** [comment]
yes please @eric-mitchell, those 5M trajectories would be very helpful if you still have them.
